File: doan_mobile/lib/models/plant_disease.dart
class PlantDisease {
  final String id;
  final String photo;
  final String name;
  final String createdAt;

  PlantDisease({
    required this.id,
    required this.photo,
    required this.name,
    required this.createdAt,
  });

  factory PlantDisease.fromJson(Map<String, dynamic> json) {
    return PlantDisease(
      id: json['id'],
      photo: json['photo'],
      name: json['name'],
      createdAt: json['createdAt'],
    );
  }
}

File: DoAn-BackEnd/AgriSystem/AgriSystem.Data/Entities/PlantDisease.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace AgriSystem.Data.Entities
{
    [Table("PlantDiseases")]
    public class PlantDisease
    {
        [Key]
        public Guid Id { get; set; }

        public string? Photo { get; set; }

        [MaxLength(255)]
        public string? Name { get; set; }

        [MaxLength(255)]
        public string? Description { get; set; }

        public bool IsDeleted { get; set; } = false;

        public Guid UserId { get; set; }

        [Required]
        [DataType(DataType.DateTime)]
        public DateTime CreatedAt { get; set; } = DateTime.Now;

        [Required]
        [DataType(DataType.DateTime)]
        public DateTime UpdatedAt { get; set; } = DateTime.Now;


        // Relation
        public virtual UserApp User { get; set; }
    }
}
